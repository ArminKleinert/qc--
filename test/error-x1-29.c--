register bits1 one;

foo() {
    continuation c (one):
}
