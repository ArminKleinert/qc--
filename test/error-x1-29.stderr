file error-x1-29.c--, line 4, characters 5-26 Error: one is not local
